2018-06-30 08:48:24.623 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 08:48:24.626 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 08:49:24.627 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 08:49:24.630 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 08:50:00.276 4197 DEBUG oslo_concurrency.lockutils [req-23353f24-28fd-436f-8d90-88f8401c3a57 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:50:01.600 4197 DEBUG neutron.agent.dhcp.agent [req-9fc122fc-19d4-44e1-85ea-a24912f5b721 - - - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:50:01.602 4197 DEBUG neutron.agent.linux.utils [req-9fc122fc-19d4-44e1-85ea-a24912f5b721 - - - - -] Unable to access /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:01.603 4197 DEBUG neutron.agent.linux.dhcp [req-9fc122fc-19d4-44e1-85ea-a24912f5b721 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-0353c366-e0cd-485e-8dea-253225082a68 on network 0353c366-e0cd-485e-8dea-253225082a68 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-06-30 08:50:01.603 4197 DEBUG neutron.agent.linux.dhcp [req-9fc122fc-19d4-44e1-85ea-a24912f5b721 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-0353c366-e0cd-485e-8dea-253225082a68 on network 0353c366-e0cd-485e-8dea-253225082a68 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-06-30 08:50:05.498 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:05.536 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:05.561 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:05.654 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:05.675 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:05.698 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', 'link', 'set', 'tap37f9246f-2f', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:05.746 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-o', 'link', 'show', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:05.796 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-30 08:50:06.115 4197 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap37f9246f-2f) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-30 08:50:06.117 4197 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-06-30 08:50:06.118 4197 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap37f9246f-2f) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-30 08:50:06.119 4197 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'37f9246f-2fac-4dd2-8b0a-d87b54953f4f', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:a5:26:b8'})), record=tap37f9246f-2f) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-30 08:50:06.173 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap37f9246f-2f', 'address', 'fa:16:3e:a5:26:b8'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.197 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.211 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap37f9246f-2f', 'netns', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.233 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', 'link', 'set', 'tap37f9246f-2f', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.249 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', 'link', 'set', 'tap37f9246f-2f', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.264 4197 DEBUG oslo_concurrency.lockutils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Acquired semaphore "iptables-qdhcp-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:50:06.266 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.289 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.328 4197 DEBUG neutron.agent.linux.iptables_manager [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-06-30 08:50:06.330 4197 DEBUG oslo_concurrency.lockutils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Releasing semaphore "iptables-qdhcp-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:50:06.331 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-o', 'link', 'show', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.350 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', 'addr', 'show', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.368 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-4', 'addr', 'add', '11.108.1.2/24', 'scope', 'global', 'dev', 'tap37f9246f-2f', 'brd', '11.108.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.379 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-4', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.394 4197 DEBUG neutron.agent.linux.dhcp [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Setting IPv4 gateway for dhcp netns on net 0353c366-e0cd-485e-8dea-253225082a68 to 11.108.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1164
2018-06-30 08:50:06.396 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.108.1.1', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.415 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-6', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.439 4197 DEBUG neutron.agent.linux.dhcp [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Cleaning stale devices for network 0353c366-e0cd-485e-8dea-253225082a68 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-06-30 08:50:06.442 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.455 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.474 4197 DEBUG neutron.agent.linux.dhcp [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:623
2018-06-30 08:50:06.476 4197 DEBUG neutron.agent.linux.dhcp [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/leases with contents:
1530427806 fa:16:3e:a5:26:b8 11.108.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:651
2018-06-30 08:50:06.477 4197 DEBUG neutron.agent.linux.dhcp [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Building host file: /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 08:50:06.478 4197 DEBUG neutron.agent.linux.dhcp [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Done building host file /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 08:50:06.481 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Unable to access /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:06.482 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host', '--addn-hosts=/var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap37f9246f-2f', '--dhcp-range=set:tag0,11.108.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:06.522 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:06.523 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:06.524 4197 DEBUG neutron.agent.linux.external_process [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:50:06.525 4197 DEBUG oslo_concurrency.lockutils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:50:06.527 4197 DEBUG oslo_concurrency.lockutils [req-23353f24-28fd-436f-8d90-88f8401c3a57 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:50:07.700 4197 DEBUG neutron.agent.dhcp.agent [req-c318a607-bee0-43e5-b9c7-b4c615bd4bae - - - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:50:07.702 4197 DEBUG neutron.agent.linux.dhcp [req-c318a607-bee0-43e5-b9c7-b4c615bd4bae - - - - -] Building host file: /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 08:50:07.724 4197 DEBUG neutron.agent.linux.dhcp [req-c318a607-bee0-43e5-b9c7-b4c615bd4bae - - - - -] Done building host file /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 08:50:07.740 4197 DEBUG neutron.agent.linux.utils [req-c318a607-bee0-43e5-b9c7-b4c615bd4bae - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '8317'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:07.804 4197 DEBUG neutron.agent.linux.dhcp [req-c318a607-bee0-43e5-b9c7-b4c615bd4bae - - - - -] Reloading allocations for network: 0353c366-e0cd-485e-8dea-253225082a68 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 08:50:07.807 4197 DEBUG neutron.agent.linux.utils [req-c318a607-bee0-43e5-b9c7-b4c615bd4bae - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-4', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:07.842 4197 DEBUG neutron.agent.linux.utils [req-c318a607-bee0-43e5-b9c7-b4c615bd4bae - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-6', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:07.888 4197 DEBUG neutron.agent.linux.utils [req-c318a607-bee0-43e5-b9c7-b4c615bd4bae - - - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:07.890 4197 DEBUG neutron.agent.linux.utils [req-c318a607-bee0-43e5-b9c7-b4c615bd4bae - - - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:07.891 4197 DEBUG neutron.agent.linux.external_process [req-c318a607-bee0-43e5-b9c7-b4c615bd4bae - - - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:50:07.892 4197 DEBUG oslo_concurrency.lockutils [req-c318a607-bee0-43e5-b9c7-b4c615bd4bae - - - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:50:07.893 4197 DEBUG oslo_concurrency.lockutils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:50:07.894 4197 DEBUG neutron.agent.dhcp.agent [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-30T06:50:02Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-0353c366-e0cd-485e-8dea-253225082a68, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4a7705d8-1b34-4de4-932a-35e812ec124f', u'ip_address': u'11.108.1.2'}], id=37f9246f-2fac-4dd2-8b0a-d87b54953f4f, mac_address=fa:16:3e:a5:26:b8, name=, network_id=0353c366-e0cd-485e-8dea-253225082a68, port_security_enabled=False, project_id=ca58ddd715c94c2c8429a568a6e39b13, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=ca58ddd715c94c2c8429a568a6e39b13, updated_at=2018-06-30T06:50:03Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:409
2018-06-30 08:50:07.895 4197 DEBUG oslo_concurrency.lockutils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:50:07.897 4197 DEBUG oslo_concurrency.lockutils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:50:07.898 4197 INFO neutron.agent.dhcp.agent [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-30T06:50:02Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-0353c366-e0cd-485e-8dea-253225082a68, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4a7705d8-1b34-4de4-932a-35e812ec124f', u'ip_address': u'11.108.1.2'}], id=37f9246f-2fac-4dd2-8b0a-d87b54953f4f, mac_address=fa:16:3e:a5:26:b8, name=, network_id=0353c366-e0cd-485e-8dea-253225082a68, port_security_enabled=False, project_id=ca58ddd715c94c2c8429a568a6e39b13, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=ca58ddd715c94c2c8429a568a6e39b13, updated_at=2018-06-30T06:50:04Z
2018-06-30 08:50:07.900 4197 DEBUG neutron.agent.dhcp.agent [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:50:07.901 4197 DEBUG neutron.agent.linux.dhcp [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Building host file: /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 08:50:07.903 4197 DEBUG neutron.agent.linux.dhcp [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Done building host file /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 08:50:07.907 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '8317'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:07.997 4197 DEBUG neutron.agent.linux.dhcp [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Reloading allocations for network: 0353c366-e0cd-485e-8dea-253225082a68 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 08:50:07.999 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-4', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:08.078 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-6', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:08.120 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:08.121 4197 DEBUG neutron.agent.linux.utils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:08.121 4197 DEBUG neutron.agent.linux.external_process [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:50:08.122 4197 DEBUG oslo_concurrency.lockutils [req-16f22638-aa5b-414f-8bd0-e57185330f03 - - - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:50:24.626 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 08:50:24.628 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 08:50:43.213 4197 DEBUG oslo_concurrency.lockutils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:50:43.215 4197 INFO neutron.agent.dhcp.agent [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-30T06:50:41Z, description=, device_id=fba7beea-649d-48cc-a5db-c165d2c998e9, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4a7705d8-1b34-4de4-932a-35e812ec124f', u'ip_address': u'11.108.1.1'}], id=7a41f47d-0c07-409e-a514-a901cb0e5a08, mac_address=fa:16:3e:2b:42:f2, name=, network_id=0353c366-e0cd-485e-8dea-253225082a68, port_security_enabled=False, project_id=ca58ddd715c94c2c8429a568a6e39b13, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=ca58ddd715c94c2c8429a568a6e39b13, updated_at=2018-06-30T06:50:41Z
2018-06-30 08:50:43.216 4197 DEBUG neutron.agent.dhcp.agent [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:50:43.217 4197 DEBUG neutron.agent.linux.dhcp [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Building host file: /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 08:50:43.222 4197 DEBUG neutron.agent.linux.dhcp [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Done building host file /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 08:50:43.228 4197 DEBUG neutron.agent.linux.utils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '8317'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:43.245 4197 DEBUG neutron.agent.linux.dhcp [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Reloading allocations for network: 0353c366-e0cd-485e-8dea-253225082a68 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 08:50:43.247 4197 DEBUG neutron.agent.linux.utils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-4', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:43.266 4197 DEBUG neutron.agent.linux.utils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-6', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:43.280 4197 DEBUG neutron.agent.linux.utils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:43.281 4197 DEBUG neutron.agent.linux.utils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:43.282 4197 DEBUG neutron.agent.linux.external_process [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:50:43.282 4197 DEBUG oslo_concurrency.lockutils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:50:44.643 4197 DEBUG oslo_concurrency.lockutils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:50:44.644 4197 INFO neutron.agent.dhcp.agent [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-30T06:50:41Z, description=, device_id=fba7beea-649d-48cc-a5db-c165d2c998e9, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4a7705d8-1b34-4de4-932a-35e812ec124f', u'ip_address': u'11.108.1.1'}], id=7a41f47d-0c07-409e-a514-a901cb0e5a08, mac_address=fa:16:3e:2b:42:f2, name=, network_id=0353c366-e0cd-485e-8dea-253225082a68, port_security_enabled=False, project_id=ca58ddd715c94c2c8429a568a6e39b13, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=ca58ddd715c94c2c8429a568a6e39b13, updated_at=2018-06-30T06:50:41Z
2018-06-30 08:50:44.645 4197 DEBUG neutron.agent.dhcp.agent [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:50:44.660 4197 DEBUG neutron.agent.linux.dhcp [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Building host file: /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 08:50:44.661 4197 DEBUG neutron.agent.linux.dhcp [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Done building host file /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 08:50:44.666 4197 DEBUG neutron.agent.linux.utils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '8317'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:44.714 4197 DEBUG neutron.agent.linux.dhcp [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Reloading allocations for network: 0353c366-e0cd-485e-8dea-253225082a68 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 08:50:44.717 4197 DEBUG neutron.agent.linux.utils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-4', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:44.736 4197 DEBUG neutron.agent.linux.utils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-6', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:44.754 4197 DEBUG neutron.agent.linux.utils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:44.755 4197 DEBUG neutron.agent.linux.utils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:44.756 4197 DEBUG neutron.agent.linux.external_process [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:50:44.757 4197 DEBUG oslo_concurrency.lockutils [req-2af85777-05cd-4e53-9e6f-bea221376f63 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:50:47.662 4197 DEBUG oslo_concurrency.lockutils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:50:47.664 4197 INFO neutron.agent.dhcp.agent [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-30T06:50:41Z, description=, device_id=fba7beea-649d-48cc-a5db-c165d2c998e9, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4a7705d8-1b34-4de4-932a-35e812ec124f', u'ip_address': u'11.108.1.1'}], id=7a41f47d-0c07-409e-a514-a901cb0e5a08, mac_address=fa:16:3e:2b:42:f2, name=, network_id=0353c366-e0cd-485e-8dea-253225082a68, port_security_enabled=False, project_id=ca58ddd715c94c2c8429a568a6e39b13, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=ca58ddd715c94c2c8429a568a6e39b13, updated_at=2018-06-30T06:50:43Z
2018-06-30 08:50:47.665 4197 DEBUG neutron.agent.dhcp.agent [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:50:47.666 4197 DEBUG neutron.agent.linux.dhcp [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Building host file: /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 08:50:47.668 4197 DEBUG neutron.agent.linux.dhcp [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Done building host file /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 08:50:47.684 4197 DEBUG neutron.agent.linux.utils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '8317'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:47.703 4197 DEBUG neutron.agent.linux.dhcp [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Reloading allocations for network: 0353c366-e0cd-485e-8dea-253225082a68 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 08:50:47.705 4197 DEBUG neutron.agent.linux.utils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-4', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:47.722 4197 DEBUG neutron.agent.linux.utils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-6', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:47.741 4197 DEBUG neutron.agent.linux.utils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:47.742 4197 DEBUG neutron.agent.linux.utils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:47.743 4197 DEBUG neutron.agent.linux.external_process [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:50:47.744 4197 DEBUG oslo_concurrency.lockutils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:50:48.976 4197 DEBUG oslo_concurrency.lockutils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:50:48.978 4197 INFO neutron.agent.dhcp.agent [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-30T06:50:41Z, description=, device_id=fba7beea-649d-48cc-a5db-c165d2c998e9, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4a7705d8-1b34-4de4-932a-35e812ec124f', u'ip_address': u'11.108.1.1'}], id=7a41f47d-0c07-409e-a514-a901cb0e5a08, mac_address=fa:16:3e:2b:42:f2, name=, network_id=0353c366-e0cd-485e-8dea-253225082a68, port_security_enabled=False, project_id=ca58ddd715c94c2c8429a568a6e39b13, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=ca58ddd715c94c2c8429a568a6e39b13, updated_at=2018-06-30T06:50:48Z
2018-06-30 08:50:48.980 4197 DEBUG neutron.agent.dhcp.agent [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:50:48.981 4197 DEBUG neutron.agent.linux.dhcp [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Building host file: /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 08:50:48.983 4197 DEBUG neutron.agent.linux.dhcp [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Done building host file /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 08:50:48.988 4197 DEBUG neutron.agent.linux.utils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '8317'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:49.016 4197 DEBUG neutron.agent.linux.dhcp [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Reloading allocations for network: 0353c366-e0cd-485e-8dea-253225082a68 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 08:50:49.016 4197 DEBUG neutron.agent.linux.utils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-4', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:49.038 4197 DEBUG neutron.agent.linux.utils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-6', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:50:49.065 4197 DEBUG neutron.agent.linux.utils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:49.066 4197 DEBUG neutron.agent.linux.utils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:50:49.067 4197 DEBUG neutron.agent.linux.external_process [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:50:49.067 4197 DEBUG oslo_concurrency.lockutils [req-1bba3d62-6731-43fa-8e96-31fbc30d5e9e - - - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:50:58.087 4197 DEBUG oslo_concurrency.lockutils [req-e601bd6a-d100-4c40-b9d6-1b4da24b5f0f b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Acquired semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:50:58.088 4197 DEBUG oslo_concurrency.lockutils [req-e601bd6a-d100-4c40-b9d6-1b4da24b5f0f b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Releasing semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:51:01.571 4197 DEBUG oslo_concurrency.lockutils [req-88a754ad-5ea1-4cc8-93d5-8562368d601e - - - - -] Acquired semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:51:01.572 4197 DEBUG oslo_concurrency.lockutils [req-88a754ad-5ea1-4cc8-93d5-8562368d601e - - - - -] Releasing semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:51:02.993 4197 DEBUG oslo_concurrency.lockutils [req-88a754ad-5ea1-4cc8-93d5-8562368d601e - - - - -] Acquired semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:51:02.994 4197 DEBUG oslo_concurrency.lockutils [req-88a754ad-5ea1-4cc8-93d5-8562368d601e - - - - -] Releasing semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:51:24.627 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 08:51:24.631 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 08:51:24.889 4197 DEBUG oslo_concurrency.lockutils [req-d57f9e75-737b-46c2-9eb3-0b4f385e8d84 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:51:24.890 4197 INFO neutron.agent.dhcp.agent [req-d57f9e75-737b-46c2-9eb3-0b4f385e8d84 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-30T06:51:23Z, description=, device_id=62309483-a60d-4c3b-b8f0-27f144a249b6, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4a7705d8-1b34-4de4-932a-35e812ec124f', u'ip_address': u'11.108.1.9'}], id=bc2868e8-3305-43f5-a14c-d0430bce8918, mac_address=fa:16:3e:1e:b0:64, name=, network_id=0353c366-e0cd-485e-8dea-253225082a68, port_security_enabled=True, project_id=ca58ddd715c94c2c8429a568a6e39b13, revision_number=3, security_groups=[u'996f2dea-0b49-4351-b7a5-f9ba537e0549'], status=DOWN, tags=[], tenant_id=ca58ddd715c94c2c8429a568a6e39b13, updated_at=2018-06-30T06:51:24Z
2018-06-30 08:51:24.892 4197 DEBUG neutron.agent.dhcp.agent [req-d57f9e75-737b-46c2-9eb3-0b4f385e8d84 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:51:24.893 4197 DEBUG neutron.agent.linux.dhcp [req-d57f9e75-737b-46c2-9eb3-0b4f385e8d84 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Building host file: /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 08:51:24.895 4197 DEBUG neutron.agent.linux.dhcp [req-d57f9e75-737b-46c2-9eb3-0b4f385e8d84 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Done building host file /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 08:51:24.899 4197 DEBUG neutron.agent.linux.utils [req-d57f9e75-737b-46c2-9eb3-0b4f385e8d84 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '8317'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:51:24.926 4197 DEBUG neutron.agent.linux.dhcp [req-d57f9e75-737b-46c2-9eb3-0b4f385e8d84 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Reloading allocations for network: 0353c366-e0cd-485e-8dea-253225082a68 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 08:51:24.928 4197 DEBUG neutron.agent.linux.utils [req-d57f9e75-737b-46c2-9eb3-0b4f385e8d84 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-4', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:51:24.947 4197 DEBUG neutron.agent.linux.utils [req-d57f9e75-737b-46c2-9eb3-0b4f385e8d84 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-6', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:51:24.970 4197 DEBUG neutron.agent.linux.utils [req-d57f9e75-737b-46c2-9eb3-0b4f385e8d84 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:51:24.971 4197 DEBUG neutron.agent.linux.utils [req-d57f9e75-737b-46c2-9eb3-0b4f385e8d84 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:51:24.972 4197 DEBUG neutron.agent.linux.external_process [req-d57f9e75-737b-46c2-9eb3-0b4f385e8d84 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:51:24.973 4197 DEBUG oslo_concurrency.lockutils [req-d57f9e75-737b-46c2-9eb3-0b4f385e8d84 b3308744f0c24e299bc4ec9ae3ef23a1 ca58ddd715c94c2c8429a568a6e39b13 - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:51:28.148 4197 DEBUG oslo_concurrency.lockutils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:51:28.149 4197 INFO neutron.agent.dhcp.agent [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-30T06:51:23Z, description=, device_id=62309483-a60d-4c3b-b8f0-27f144a249b6, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4a7705d8-1b34-4de4-932a-35e812ec124f', u'ip_address': u'11.108.1.9'}], id=bc2868e8-3305-43f5-a14c-d0430bce8918, mac_address=fa:16:3e:1e:b0:64, name=, network_id=0353c366-e0cd-485e-8dea-253225082a68, port_security_enabled=True, project_id=ca58ddd715c94c2c8429a568a6e39b13, revision_number=4, security_groups=[u'996f2dea-0b49-4351-b7a5-f9ba537e0549'], status=DOWN, tags=[], tenant_id=ca58ddd715c94c2c8429a568a6e39b13, updated_at=2018-06-30T06:51:26Z
2018-06-30 08:51:28.150 4197 DEBUG neutron.agent.dhcp.agent [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:51:28.152 4197 DEBUG neutron.agent.linux.dhcp [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Building host file: /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 08:51:28.162 4197 DEBUG neutron.agent.linux.dhcp [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Done building host file /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 08:51:28.171 4197 DEBUG neutron.agent.linux.utils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Running command (rootwrap daemon): ['kill', '-HUP', '8317'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:51:28.186 4197 DEBUG neutron.agent.linux.dhcp [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Reloading allocations for network: 0353c366-e0cd-485e-8dea-253225082a68 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 08:51:28.195 4197 DEBUG neutron.agent.linux.utils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-4', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:51:28.209 4197 DEBUG neutron.agent.linux.utils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-6', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:51:28.222 4197 DEBUG neutron.agent.linux.utils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:51:28.223 4197 DEBUG neutron.agent.linux.utils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:51:28.224 4197 DEBUG neutron.agent.linux.external_process [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:51:28.226 4197 DEBUG oslo_concurrency.lockutils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:51:28.906 4197 DEBUG oslo_concurrency.lockutils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:51:28.907 4197 INFO neutron.agent.dhcp.agent [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-30T06:51:23Z, description=, device_id=62309483-a60d-4c3b-b8f0-27f144a249b6, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4a7705d8-1b34-4de4-932a-35e812ec124f', u'ip_address': u'11.108.1.9'}], id=bc2868e8-3305-43f5-a14c-d0430bce8918, mac_address=fa:16:3e:1e:b0:64, name=, network_id=0353c366-e0cd-485e-8dea-253225082a68, port_security_enabled=True, project_id=ca58ddd715c94c2c8429a568a6e39b13, revision_number=5, security_groups=[u'996f2dea-0b49-4351-b7a5-f9ba537e0549'], status=DOWN, tags=[], tenant_id=ca58ddd715c94c2c8429a568a6e39b13, updated_at=2018-06-30T06:51:28Z
2018-06-30 08:51:28.909 4197 DEBUG neutron.agent.dhcp.agent [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:51:28.911 4197 DEBUG neutron.agent.linux.dhcp [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Building host file: /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 08:51:28.914 4197 DEBUG neutron.agent.linux.dhcp [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Done building host file /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 08:51:28.919 4197 DEBUG neutron.agent.linux.utils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Running command (rootwrap daemon): ['kill', '-HUP', '8317'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:51:28.935 4197 DEBUG neutron.agent.linux.dhcp [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Reloading allocations for network: 0353c366-e0cd-485e-8dea-253225082a68 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 08:51:28.937 4197 DEBUG neutron.agent.linux.utils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-4', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:51:28.951 4197 DEBUG neutron.agent.linux.utils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-6', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:51:28.974 4197 DEBUG neutron.agent.linux.utils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:51:28.976 4197 DEBUG neutron.agent.linux.utils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:51:28.976 4197 DEBUG neutron.agent.linux.external_process [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:51:28.977 4197 DEBUG oslo_concurrency.lockutils [req-2122164b-706b-41a2-a185-1ac453d63985 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:52:14.179 4197 DEBUG oslo_concurrency.lockutils [req-0e552d06-adae-432d-90e7-088e06e3ebed c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:52:14.180 4197 DEBUG oslo_concurrency.lockutils [req-0e552d06-adae-432d-90e7-088e06e3ebed c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Releasing semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:52:16.282 4197 DEBUG oslo_concurrency.lockutils [req-0e552d06-adae-432d-90e7-088e06e3ebed c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:52:16.284 4197 DEBUG oslo_concurrency.lockutils [req-0e552d06-adae-432d-90e7-088e06e3ebed c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Releasing semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:52:24.628 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 08:52:24.633 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 08:53:24.641 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 08:53:24.643 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 08:54:24.645 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 08:54:24.650 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 08:55:24.650 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 08:55:24.655 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.006s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 08:56:24.650 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 08:56:24.654 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 08:57:24.650 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 08:57:24.660 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.009s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 08:57:33.479 4197 DEBUG oslo_concurrency.lockutils [req-67cf3cac-c94b-4632-b233-deec42e08160 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:57:33.480 4197 DEBUG neutron.agent.dhcp.agent [req-67cf3cac-c94b-4632-b233-deec42e08160 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:57:33.482 4197 DEBUG neutron.agent.linux.utils [req-67cf3cac-c94b-4632-b233-deec42e08160 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'dhcp_release', 'tap37f9246f-2f', '11.108.1.1', 'fa:16:3e:2b:42:f2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:57:33.499 4197 DEBUG neutron.agent.linux.dhcp [req-67cf3cac-c94b-4632-b233-deec42e08160 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Building host file: /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 08:57:33.503 4197 DEBUG neutron.agent.linux.dhcp [req-67cf3cac-c94b-4632-b233-deec42e08160 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Done building host file /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 08:57:33.507 4197 DEBUG neutron.agent.linux.utils [req-67cf3cac-c94b-4632-b233-deec42e08160 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '8317'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:57:33.517 4197 DEBUG neutron.agent.linux.dhcp [req-67cf3cac-c94b-4632-b233-deec42e08160 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Reloading allocations for network: 0353c366-e0cd-485e-8dea-253225082a68 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 08:57:33.519 4197 DEBUG neutron.agent.linux.utils [req-67cf3cac-c94b-4632-b233-deec42e08160 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-4', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:57:33.530 4197 DEBUG neutron.agent.linux.utils [req-67cf3cac-c94b-4632-b233-deec42e08160 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68', 'ip', '-6', 'route', 'list', 'dev', 'tap37f9246f-2f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:57:33.544 4197 DEBUG neutron.agent.linux.utils [req-67cf3cac-c94b-4632-b233-deec42e08160 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:57:33.545 4197 DEBUG neutron.agent.linux.utils [req-67cf3cac-c94b-4632-b233-deec42e08160 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:57:33.546 4197 DEBUG neutron.agent.linux.external_process [req-67cf3cac-c94b-4632-b233-deec42e08160 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:57:33.547 4197 DEBUG oslo_concurrency.lockutils [req-67cf3cac-c94b-4632-b233-deec42e08160 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:57:58.356 4197 DEBUG oslo_concurrency.lockutils [req-6687839b-906f-4244-9b63-cff156077b21 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:57:58.358 4197 DEBUG neutron.agent.dhcp.agent [req-6687839b-906f-4244-9b63-cff156077b21 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:57:58.361 4197 DEBUG neutron.agent.linux.utils [req-6687839b-906f-4244-9b63-cff156077b21 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['kill', '-9', '8317'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:57:58.381 4197 DEBUG neutron.agent.linux.utils [req-6687839b-906f-4244-9b63-cff156077b21 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-30 08:57:58.416 4197 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tap37f9246f-2f) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-30 08:57:58.524 4197 DEBUG neutron.agent.linux.utils [req-410225aa-4652-4194-be64-5c77555d8944 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:57:58.536 4197 DEBUG neutron.agent.linux.utils [req-410225aa-4652-4194-be64-5c77555d8944 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-0353c366-e0cd-485e-8dea-253225082a68'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:57:58.559 4197 DEBUG oslo_concurrency.lockutils [req-410225aa-4652-4194-be64-5c77555d8944 - - - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:58:08.542 4197 DEBUG oslo_concurrency.lockutils [req-0d8ce1c8-1444-45be-b81f-6cfa219f3fa1 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:58:08.543 4197 DEBUG neutron.agent.dhcp.agent [req-0d8ce1c8-1444-45be-b81f-6cfa219f3fa1 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:58:08.545 4197 DEBUG neutron.agent.linux.dhcp [req-0d8ce1c8-1444-45be-b81f-6cfa219f3fa1 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Impossibile accedere a /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:268
2018-06-30 08:58:08.545 4197 DEBUG neutron.agent.linux.dhcp [req-0d8ce1c8-1444-45be-b81f-6cfa219f3fa1 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Agent does not have an interface on this network anymore, skipping reload: 0353c366-e0cd-485e-8dea-253225082a68 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:506
2018-06-30 08:58:08.547 4197 DEBUG neutron.agent.linux.utils [req-0d8ce1c8-1444-45be-b81f-6cfa219f3fa1 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:58:08.548 4197 DEBUG neutron.agent.linux.utils [req-0d8ce1c8-1444-45be-b81f-6cfa219f3fa1 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:58:08.548 4197 DEBUG neutron.agent.linux.external_process [req-0d8ce1c8-1444-45be-b81f-6cfa219f3fa1 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:58:08.549 4197 DEBUG oslo_concurrency.lockutils [req-0d8ce1c8-1444-45be-b81f-6cfa219f3fa1 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:58:16.713 4197 DEBUG oslo_concurrency.lockutils [req-f0fbfaa6-f1f2-4823-ba74-dbe1173546d2 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:58:17.029 4197 DEBUG neutron.agent.linux.utils [req-b872e6c3-03f0-4761-b279-d8fa444571ed - - - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:58:17.030 4197 DEBUG neutron.agent.linux.utils [req-b872e6c3-03f0-4761-b279-d8fa444571ed - - - - -] Unable to access /var/lib/neutron/external/pids/0353c366-e0cd-485e-8dea-253225082a68.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:58:17.030 4197 DEBUG neutron.agent.linux.external_process [req-b872e6c3-03f0-4761-b279-d8fa444571ed - - - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:58:17.031 4197 DEBUG neutron.agent.dhcp.agent [req-b872e6c3-03f0-4761-b279-d8fa444571ed - - - - -] Calling driver for network: 0353c366-e0cd-485e-8dea-253225082a68 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 08:58:17.032 4197 DEBUG neutron.agent.linux.utils [req-b872e6c3-03f0-4761-b279-d8fa444571ed - - - - -] Unable to access /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:58:17.032 4197 DEBUG neutron.agent.linux.utils [req-b872e6c3-03f0-4761-b279-d8fa444571ed - - - - -] Unable to access /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 08:58:17.033 4197 DEBUG neutron.agent.linux.external_process [req-b872e6c3-03f0-4761-b279-d8fa444571ed - - - - -] No process started for 0353c366-e0cd-485e-8dea-253225082a68 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 08:58:17.034 4197 DEBUG neutron.agent.linux.dhcp [req-b872e6c3-03f0-4761-b279-d8fa444571ed - - - - -] Impossibile accedere a /var/lib/neutron/dhcp/0353c366-e0cd-485e-8dea-253225082a68/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:268
2018-06-30 08:58:17.034 4197 DEBUG neutron.agent.linux.dhcp [req-b872e6c3-03f0-4761-b279-d8fa444571ed - - - - -] No interface exists for network 0353c366-e0cd-485e-8dea-253225082a68 destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1482
2018-06-30 08:58:17.065 4197 DEBUG neutron.agent.linux.utils [req-a4405363-100e-45f0-95a5-8444fe64e580 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 08:58:17.079 4197 DEBUG neutron.agent.linux.dhcp [req-a4405363-100e-45f0-95a5-8444fe64e580 - - - - -] Namespace already deleted: qdhcp-0353c366-e0cd-485e-8dea-253225082a68 _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:246
2018-06-30 08:58:17.081 4197 DEBUG oslo_concurrency.lockutils [req-a4405363-100e-45f0-95a5-8444fe64e580 - - - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:58:18.521 4197 DEBUG oslo_concurrency.lockutils [req-f0fbfaa6-f1f2-4823-ba74-dbe1173546d2 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 08:58:18.522 4197 DEBUG oslo_concurrency.lockutils [req-f0fbfaa6-f1f2-4823-ba74-dbe1173546d2 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Releasing semaphore "dhcp-agent-network-lock-0353c366-e0cd-485e-8dea-253225082a68" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 08:58:21.364 4197 DEBUG neutron.agent.dhcp.agent [req-04f4944e-ea31-4622-a46f-1f16818cc322 - - - - -] resync (0353c366-e0cd-485e-8dea-253225082a68): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:255
2018-06-30 08:58:21.365 4197 INFO neutron.agent.dhcp.agent [req-04f4944e-ea31-4622-a46f-1f16818cc322 - - - - -] Synchronizing state
2018-06-30 08:58:21.974 4197 INFO neutron.agent.dhcp.agent [req-08982b0d-e894-4f04-8b76-dbeb721d0754 - - - - -] All active networks have been fetched through RPC.
2018-06-30 08:58:21.975 4197 INFO neutron.agent.dhcp.agent [-] Starting network 3a89a4af-574f-492a-9eb4-eae63d51f625 dhcp configuration
2018-06-30 08:58:21.976 4197 INFO neutron.agent.dhcp.agent [-] Finished network 3a89a4af-574f-492a-9eb4-eae63d51f625 dhcp configuration
2018-06-30 08:58:21.976 4197 INFO neutron.agent.dhcp.agent [req-08982b0d-e894-4f04-8b76-dbeb721d0754 - - - - -] Synchronizing state complete
2018-06-30 08:58:24.651 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 08:58:24.657 4197 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.007s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
